azure-terraform/terraform/cosmos-db.tf

```diff
@@ -24,7 +24,6 @@ resource "azurerm_cosmosdb_account" "db" {
   }
 
   geo_location {
-    prefix            = "porterform-${azurerm_resource_group.rg.location}"
```

Comment:
Why was this removed?

Reply:
The property is now deprecated from the azurerm provider as the API  no longer accepts it